Bylaws.md

```diff
@@ -50,10 +56,12 @@ Categories of Forum Membership are defined as follows:
 
 * Voting Member, which contains the following sub-categories:
    * Certificate Issuer: The member organization operates a certification authority that meets the requirements for membership as a Certificate Issuer in one or more CWGs.
-   * Root Certificate Issuer: The member organization operates a certification authority that meets the requirements for membership as a Root Certificate Issuer in one or more CWGs.
    * Certificate Consumer: The member organization meets the requirements for membership as a Certificate Consumer in one or more CWGs.
 * Associate Member: As defined in Section 3.1
 * Interested Party: As defined in Section 3.2
+* Probational Certificate Issuer: A Certificate Issuer applying for membership but awaiting full qualification under Section 2.1. During the probational state, the organization has no voting rights. 
```

Comment:
Do we want / need to set any limits on how long a CA can stay on as a Probational Certificate Issuer?

Reply:
We could but the specifics could be set in a CWG Charter. Should we suggest 1y with the option to renew? I recall some past discussions where we said we would consider all Members of that category for renewal during one of the 3 annual F2F meetings.

Reply:
Addressed in a9c4b4b47c8955d9d4fdf671e721cbaaf896e843